src/boots/boots.gd

```diff
@@ -0,0 +1,45 @@
+extends Node2D
+
+
+var sequence_to_reach_golden_piece: Array[String] = ["down","down","down","down","down"]
+var past_moves_relevant_to_code: Array[String] = ["","","","",""]
+
+# Called when the node enters the scene tree for the first time.
+
+func _ready() -> void:
+	print("BOOTS ON!!!")
+	connect_move_signal_to_player()
+
+func connect_move_signal_to_player() -> void:
+	var parent_player_node = get_node("../")
+	# Yeah!! You can get stuff 2 parents up the tree get_node("../../")
+	if parent_player_node != self:
+		parent_player_node.made_successful_move.connect(Callable(self, "new_move_made"))
```

Comment:
you don't need to make a new callable object? functions are callables by default, 
parent_player_node.made_successful_move.connect(new_move_made)